clang/test/CodeGen/aarch64-sme2-intrinsics/acle_sme2_fmlas16.c

```diff
@@ -0,0 +1,592 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 4
```

Comment:
If you change these prototypes to be sme2 like the ACLE we need to change these run lines to be sme2